### ISSUE TITLE
Release PR created with repeated commits and same version number

### ISSUE THREAD

**Issue by mwjaco** [google-github-actions/release-please-action#481]
### TL;DR

My private repo recently moved from running Release Please in Google Cloud CLI to using the Github Action. The Release Please Github Action created its first release PR correctly--`v66.16.9`--but now it's stuck.

After merging the release PR for `v66.16.0`, the next conventional commit led it to create another `v66.16.0` PR with the same commits as the original `v66.16.0` release.

### Expected behavior

After merging the release PR, the next conventional commit should create a new release PR with a new version number. The changelog should not include previous commits to the new release PR.

### Observed behavior

The previous release PR's changelog is duplicated in the next releaes PR's changelog and the version number doesn't change.

### Action YAML

```yaml
on:
  push:
    branches:
      - dev
name: release-please
jobs:
  release-please:
    runs-on: ubuntu-latest
    steps:
      - uses: google-github-actions/release-please-action@v3
        with:
          changelog-types: '[{ "type": "feat", "section": "Features" },{ "type": "fix", "section": "Bug Fixes" },{ "type": "perf", "section": "Performance Improvements" },{ "type": "deps", "section": "Dependencies" },{ "type": "revert", "section": "Reverts" },{ "type": "docs", "section": "Documentation" },{ "type": "style", "section": "Styles" },{ "type": "chore", "section": "Miscellaneous Chores" },{ "type": "refactor", "section": "Code Refactoring" },{ "type": "test", "section": "Tests" },{ "type": "build", "section": "Build System" },{ "type": "ci", "section": "Continuous Integration" }]'
          command: release-pr
          default-branch: dev
          package-name: release-please-action
          pull-request-title-pattern: 'chore: release ${version}'
          release-type: node
```

### Log output

```text
✔ Looking for latest release on branch: dev with prefix: 
⚠ pullRequestTitlePattern miss the part of '${scope}'
⚠ pullRequestTitlePattern miss the part of '${component}'
⚠ pullRequestTitlePattern miss the part of '${scope}'
⚠ pullRequestTitlePattern miss the part of '${component}'
⚠ pullRequestTitlePattern miss the part of '${scope}'
⚠ pullRequestTitlePattern miss the part of '${component}'
⚠ pullRequestTitlePattern miss the part of '${scope}'
⚠ pullRequestTitlePattern miss the part of '${component}'
⚠ pullRequestTitlePattern miss the part of '${scope}'
⚠ pullRequestTitlePattern miss the part of '${component}'
⚠ pullRequestTitlePattern miss the part of '${scope}'
⚠ pullRequestTitlePattern miss the part of '${component}'
✔ Building pull requests
✔ Building strategies by path
✔ Collecting release commit SHAs
⚠ Found release tag with component 'v', but not configured in manifest
⚠ Found release tag with component 'v', but not configured in manifest
⚠ Found release tag with component 'v', but not configured in manifest
⚠ Found release tag with component 'v', but not configured in manifest
⚠ Found release tag with component 'v', but not configured in manifest
⚠ Found release tag with component 'v', but not configured in manifest
⚠ Found release tag with component 'v', but not configured in manifest
⚠ Found release tag with component 'v', but not configured in manifest
⚠ Found release tag with component 'v', but not configured in manifest
⚠ Found release tag with component 'v', but not configured in manifest
⚠ Found release tag with component 'v', but not configured in manifest
⚠ Found release tag with component 'v', but not configured in manifest
⚠ Found release tag with component 'v', but not configured in manifest
⚠ Found release tag with component 'v', but not configured in manifest
⚠ Found release tag with component 'v', but not configured in manifest
⚠ Found release tag with component 'v', but not configured in manifest
⚠ Found release tag with component 'v', but not configured in manifest
⚠ Found release tag with component 'v', but not configured in manifest
⚠ Found release tag with component 'v', but not configured in manifest
⚠ Found release tag with component 'v', but not configured in manifest
⚠ Found release tag with component 'v', but not configured in manifest
⚠ Found release tag with component 'v', but not configured in manifest
⚠ Found release tag with component 'v', but not configured in manifest
⚠ Found release tag with component 'v', but not configured in manifest
⚠ Found release tag with component 'v', but not configured in manifest
⚠ Found release tag with component 'v', but not configured in manifest
⚠ Found release tag with component 'v', but not configured in manifest
⚠ Found release tag with component 'v', but not configured in manifest
⚠ Found release tag with component 'v', but not configured in manifest
⚠ Found release tag with component 'v', but not configured in manifest
⚠ Found release tag with component 'v', but not configured in manifest
⚠ Found release tag with component 'v', but not configured in manifest
⚠ Found release tag with component 'v', but not configured in manifest
⚠ Found release tag with component 'v', but not configured in manifest
⚠ Found release tag with component 'v', but not configured in manifest
⚠ Found release tag with component 'v', but not configured in manifest
⚠ Found release tag with component 'v', but not configured in manifest
⚠ Found release tag with component 'v', but not configured in manifest
⚠ Found release tag with component 'v', but not configured in manifest
⚠ Found release tag with component 'v', but not configured in manifest
⚠ Found release tag with component 'v', but not configured in manifest
⚠ Found release tag with component 'v', but not configured in manifest
⚠ Found release tag with component 'v', but not configured in manifest
⚠ Found release tag with component 'v', but not configured in manifest
⚠ Found release tag with component 'v', but not configured in manifest
⚠ Found release tag with component 'v', but not configured in manifest
⚠ Found release tag with component 'v', but not configured in manifest
⚠ Found release tag with component 'v', but not configured in manifest
⚠ Found release tag with component 'v', but not configured in manifest
⚠ Found release tag with component 'v', but not configured in manifest
⚠ Found release tag with component 'v', but not configured in manifest
⚠ Found release tag with component 'v', but not configured in manifest
⚠ Found release tag with component 'v', but not configured in manifest
⚠ Found release tag with component 'v', but not configured in manifest
⚠ Found release tag with component 'v', but not configured in manifest
⚠ Found release tag with component 'v', but not configured in manifest
⚠ Found release tag with component 'v', but not configured in manifest
⚠ Found release tag with component 'v', but not configured in manifest
⚠ Found release tag with component 'v', but not configured in manifest
⚠ Found release tag with component 'v', but not configured in manifest
⚠ Found release tag with component 'v', but not configured in manifest
⚠ Found release tag with component 'v', but not configured in manifest
⚠ Found release tag with component 'v', but not configured in manifest
⚠ Found release tag with component 'v', but not configured in manifest
⚠ Found release tag with component 'v', but not configured in manifest
⚠ Found release tag with component 'v', but not configured in manifest
⚠ Found release tag with component 'v', but not configured in manifest
⚠ Found release tag with component 'v', but not configured in manifest
⚠ Found release tag with component 'v', but not configured in manifest
⚠ Found release tag with component 'v', but not configured in manifest
⚠ Found release tag with component 'v', but not configured in manifest
⚠ Found release tag with component 'v', but not configured in manifest
⚠ Found release tag with component 'v', but not configured in manifest
⚠ Found release tag with component 'v', but not configured in manifest
⚠ Found release tag with component 'v', but not configured in manifest
⚠ Found release tag with component 'v', but not configured in manifest
⚠ Found release tag with component 'v', but not configured in manifest
⚠ Found release tag with component 'v', but not configured in manifest
⚠ Found release tag with component 'v', but not configured in manifest
⚠ Found release tag with component 'v', but not configured in manifest
⚠ Found release tag with component 'v', but not configured in manifest
⚠ Found release tag with component 'v', but not configured in manifest
⚠ Found release tag with component 'v', but not configured in manifest
⚠ Found release tag with component 'v', but not configured in manifest
⚠ Found release tag with component 'v', but not configured in manifest
⚠ Found release tag with component 'v', but not configured in manifest
⚠ Found release tag with component 'v', but not configured in manifest
⚠ Found release tag with component 'v', but not configured in manifest
⚠ Found release tag with component 'v', but not configured in manifest
⚠ Found release tag with component 'v', but not configured in manifest
⚠ Found release tag with component 'v', but not configured in manifest
⚠ Found release tag with component 'v', but not configured in manifest
⚠ Found release tag with component 'v', but not configured in manifest
⚠ Found release tag with component 'v', but not configured in manifest
⚠ Found release tag with component 'v', but not configured in manifest
⚠ Found release tag with component 'v', but not configured in manifest
⚠ Found release tag with component 'v', but not configured in manifest
⚠ Found release tag with component 'v', but not configured in manifest
⚠ Unable to parse release name: Emergency revert switch
⚠ Found release tag with component 'v', but not configured in manifest
⚠ Found release tag with component '20', but not configured in manifest
⚠ Found release tag with component '20', but not configured in manifest
⚠ Found release tag with component '20', but not configured in manifest
⚠ Expected 1 releases, only found 0
⚠ Missing 1 paths: .
✔ Collecting commits since all latest releases
✔ PR #9947 has many files, backfilling
✔ PR #10041 has many files, backfilling
✔ PR #10023 has many files, backfilling
✔ PR #10010 has many files, backfilling
✔ Splitting 500 commits by path
✔ Building candidate release pull request for path: .
✔ Considering: 18 commits
⚠ pullRequestTitlePattern miss the part of '${scope}'
⚠ pullRequestTitlePattern miss the part of '${component}'
✔ Looking for open release pull requests
✔ found 0 open release pull requests.
✔ Looking for snoozed release pull requests
✔ found 0 snoozed release pull requests.
✔ updating from 66.16.0 to 66.16.0
⚠ file npm-shrinkwrap.json did not exist
⚠ file samples/package.json did not exist
✔ updating from 66.16.0 to 66.16.0
⚠ file .release-please-manifest.json did not exist
✔ Starting GitHub PR workflow...
✔ Successfully found branch HEAD sha "cf5a4faba3d65a59ee3cd815f18e7088bcdc667f".
✔ Skipping branch creation step...
✔ Got the latest commit tree
✔ Successfully created a tree with the desired changes with SHA 3b85c209b604b374dacfee10e7b3e085ea3e650c
✔ Successfully created commit. See commit at https://api.github.com/repos/<redacted>
✔ Successfully updated reference release-please--branches--dev to d2a4456f4b3d9b7db9c9be476da5cfac1b181b95
✔ Successfully opened pull request available at url: https://api.github.com/repos/<redacted>.
✔ Successfully opened pull request: 10375.
✔ Successfully added labels autorelease: pending to issue: 10375
```

### Additional information

It seems to be a combination of https://github.com/google-github-actions/release-please-action/issues/268 and https://github.com/google-github-actions/release-please-action/issues/292

In https://github.com/google-github-actions/release-please-action/issues/292 it's recommended to use "Squash & Merge," but we already do that and the problem persists anyway.

**Comment by targos** [comment]
I have the same issue (also in a private repository). @mwjaco did you find a workaround?

**Comment by targos** [comment]
The bug seems fixed in the latest version. We just pushed a new change to our repo and the release PR was updated and was correct.
